riocli/auth/token.py

```diff
@@ -46,11 +46,11 @@ def token(email: str, password: str, level: int = 0):
     if not email:
         email = config.data.get("email_id", None)
 
-    if not password:
-        password = config.data.get("password", None)
+    password = password or click.prompt('Password', hide_input=True)
```

Comment:
thought: I believe this will also be a breaking change. There will be scripts depending on password being there to generate the token.